core/worker/src/main/java/com/cognifide/aet/worker/drivers/chrome/ChromeWebDriverFactory.java

```diff
@@ -52,37 +37,28 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.metatype.annotations.Designate;
+
 
-@Service
 @Component(
-    policy = ConfigurationPolicy.REQUIRE,
-    description = "AET Chrome WebDriver Factory",
-    label = "AET Chrome WebDriver Factory",
-    metatype = true)
-@Properties({@Property(name = Constants.SERVICE_VENDOR, value = "Cognifide Ltd")})
+    configurationPolicy = ConfigurationPolicy.REQUIRE,
+    property = {"name = " + Constants.SERVICE_VENDOR, "value = Cognifide Ltd"}
+)
+@Designate(ocd = ChromeWebDriverFactoryConf.class)
 public class ChromeWebDriverFactory implements WebDriverFactory {
 
-  private static final String DEFAULT_BROWSER_NAME = "chrome";
-
   @Reference
   private HttpRequestExecutorFactory requestExecutorFactory;
 
-  @Property(name = NAME,
-      label = NAME_LABEL,
-      description = NAME_DESC,
-      value = DEFAULT_BROWSER_NAME)
-  private String name;
-
-  @Property(name = SELENIUM_GRID_URL,
-      label = SELENIUM_GRID_URL_LABEL,
-      description = "Url to selenium grid hub. When null local Chrome driver will be used. Local Chrome driver does not work on Linux",
-      value = DEFAULT_SELENIUM_GRID_URL)
-  private String seleniumGridUrl;
+  private ChromeWebDriverFactoryConf chromeWebDriverFactoryConf;
```

Comment:
In PRs for other modules I've seen convention of naming those properties `config`. Could we please keep it also here?